packages/lib-sourcify/src/Compilation/CompilationTypes.ts

```diff
@@ -125,3 +126,15 @@ export interface CompilationTarget {
 }
 
 export type CompilationLanguage = 'Solidity' | 'Vyper';
+
+export type CompilationErrorCode =
+  | 'cannot_generate_cbor_auxdata_positions'
+  | 'invalid_compiler_version'
+  | 'contract_not_found_in_compiler_output'
```

Comment:
I don't see this error code used anywhere. Also, there are still usual `Error` instances thrown in the compilation classes. Will you change that later?

Reply:
> . Also, there are still usual Error instances thrown in the compilation classes. Will you change that later?

I kept standard Errors because they are not actually CompilationError, and I thought it would be confusing to throw them as CompilationError. What do you think?

Reply:
ah right these are the getter functions. Then it's fine